docs/notebooks/Llama_Stack_Building_AI_Applications.ipynb

```diff
@@ -0,0 +1,5064 @@
+      "id": "d119026e",
+      "metadata": {
+        "id": "d119026e",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "cfd15b9f-c612-4924-dda7-eeace461a94b"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "User> Write me a sonnet about llama green\n",
+            "Assistant> In Andes high, where snow-capped peaks do stand,\n",
+            "A creature roams, with gentle, peaceful eye,\n",
+            "The llama, furry, soft, with woolly hand,\n",
+            "A mystic beauty, in the mountains' sigh.\n",
+            "\n",
+            "Its long neck bends, its ears, a delicate curl,\n",
+            "As it ascends the steep and winding trails,\n",
+            "Its soft humming sounds, a soothing twirl,\n",
+            "A soothing balm, for weary mountaineer's tales.\n",
+            "\n",
+            "But when it walks, its paws, with quiet pace,\n",
+            "Mark out a path, through rocks and dusty space,\n",
+            "And when it stops, its gentle eyes, with grace,\n",
+            "Seem to hold a wisdom, in its peaceful face.\n",
+            "\n",
+            "And as we look, upon this wondrous sight,\n",
+            "We find ourselves, in awe, of nature's light.\n",
+            "\n",
+            "Note: This sonnet follows the traditional Shakespearean sonnet structure, with 14 lines, and a rhyme scheme of ABAB CDCD EFEF GG.\n"
+          ]
+        }
+      ],
+      "source": [
+        "from llama_stack_client.lib.inference.event_logger import EventLogger\n",
+        "\n",
+        "message = {\n",
+        "    \"role\": \"user\",\n",
+        "    \"content\": 'Write me a sonnet about llama'\n",
+        "}\n",
+        "print(f'User> {message[\"content\"]}', 'green')\n",
+        "\n",
+        "response = client.inference.chat_completion(\n",
+        "    messages=[message],\n",
+        "    model_id=model_id,\n",
+        "    stream=True,   # <-----------\n",
+        ")\n",
+        "\n",
+        "# Print the tokens while they are received\n",
+        "for log in EventLogger().log(response):\n",
+        "    log.print()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 2.0. Strcutured Decoding\n",
+        "- You may use `response_format` to get a JSON structured output from the model."
+      ],
+      "metadata": {
+        "id": "OmU6Dr9zBiGM"
+      },
+      "id": "OmU6Dr9zBiGM"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from pydantic import BaseModel\n",
+        "from rich.pretty import pprint\n",
+        "\n",
+        "class Output(BaseModel):\n",
+        "    name: str\n",
+        "    year_born: str\n",
+        "    year_retired: str\n",
+        "\n",
+        "user_input = \"Michael Jordan was born in 1963. He played basketball for the Chicago Bulls. He retired in 2003.\"\n",
```

Comment:
I think you could add a `Extract this information into JSON for me." instruction at the end of the user input.